starfish/test/test_config.py

```diff
@@ -50,3 +52,49 @@ def test_lookup_deep():
     with raises(AttributeError):
         config.lookup(["a", "b", "c", "d"])
     assert config.lookup(["a", "b", "c", "d"], "x") == "x"
+
+
+def test_cache_config():
+    config = Config("""{
+        "cache": {
+             "enabled": true,
+             "size_limit": 5e9,
+             "directory": "/tmp"
+         }
+    }""")
+    cache_config = config.lookup(["cache"], {})
+    assert cache_config["enabled"]
+    assert cache_config["size_limit"] == 5 * 10 ** 9
+
+
+@mark.parametrize("name,config", (
+    ("enabled", {
+        "expected": 69688,
+        "backend": {
+            "cache": {
+                "enabled": True,
+            }}}),
+    ("disabled", {
+        "expected": 0,
+        "backend": {
+            "cache": {
+                "enabled": False,
+            }}}),
```

Comment:
would be super cool to have a test case for a cache that's enabled but size-limited.

Reply:
will do.

Reply:
test added.